src/store/create-store-module.js

```diff
@@ -45,9 +45,12 @@ export default (oidcSettings, storeSettings = {}, oidcEventListeners = {}) => {
   }
 
   const isAuthenticated = (state) => {
-    if (state.id_token) {
+    let isAuthenticatedBy = oidcConfig['isAuthenticatedBy'] || 'id_token'
```

Comment:
Also, I would prefer to have this configuration in `storeSettings` rarher than `oidcConfig` since it does not directly change the behaviour of the open id implementation. Since the docs of this project could be improved I understand that this is not super obvious.

To be consistent with other settings I would suggest to add it in the top of the store module factory (line 8):

```
storeSettings = objectAssign([
    { 
      namespaced: false,
      isAuthenticatedBy: 'id_token'
    },
    storeSettings
  ])
```

Reply:
Sure, it felt a bit wrong anyway. Happy to hear there's a better place!